benchmarks/runners/apfel_bench.py

```diff
@@ -122,12 +114,7 @@ def benchmark_ic(self):
 
         th = self.ffns_theory.copy()
         th.update(
-            {
-                "PTO": [1],
-                "IC": [1],
-                "mc": [1.4142135623730951, 2.0],
-                "Qmc": [1.4142135623730951, 2.0],
-            }
+            {"PTO": [1], "IC": [1], "mc": [2.0], "Qmc": [2.0],}
```

Comment:
- ``Qmc`` should not be used
- if ``IC`` is working then it is because it doesn't have an effect, so maybe we should remove it as well

Reply:
Okay, yes it's used by apfel but only when HQ is MSBAR